src/BlazorWebView/tests/MauiDeviceTests/ApplicationStub.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Maui.Controls;
```

Comment:
This file and the next few files (AppStub, ControlHandlerTestBase, etc.) are copied from the MAUI Controls DeviceTests, but I then removed anything that wasn't needed by the BlazorWebView tests (it was too hard to get it to compile, and too hard to directly reuse the types).